src/tests/multihost/alltests/test_ldap_password_policy.py

```diff
@@ -103,7 +105,7 @@ def test_bz748856(multihost, backupsssdconf, common_sssd_setup):
             4. clear sssd logs and cache
         :steps:
             1. The user is able to authenticate using SSH password by asserting the
-                output of the tools.auth_client_ssh_password() function.
+                output of the check_login_client function.
```

Comment:
Its a doc string error , here i am using new ssh module so doc string must change . Fixed now

Reply:
Hi,

thanks for the update. But how does the new ssh module indicate errors? By exception or dies it return an error code?

bye,
Sumit

Reply:
It will rise an exception, For example:

(Pdb) check_login_client(multihost, 'ppuser1', 'Secret123')    <---  with correct password
(Pdb) check_login_client(multihost, 'ppuser1', 'Secret1234')  <--- with wrong password
*** ssh2.exceptions.AuthenticationError
(Pdb)